Plugins/Locales/itIT.lua

```diff
@@ -190,7 +190,7 @@ L.blockTalkingHeadTimewalking = "Viaggi nel Tempo (Spedizioni & Incursioni)"
 L.blockTalkingHeadScenarios = "Scenari"
 
 L.redirectPopups = "Reindirizza i banner popup nelle su bigwigs"
---L.redirectPopupsDesc = "Popup banners in the middle of your screen such as the '|cFF33FF99vault slot unlocked|r' banner will instead be displayed as BigWigs messages. Questi banner possono essere abbastanza grandi, durare molto, e bloccare la possibilità di cliccarci attraverso."
+L.redirectPopupsDesc = "I banner popup in mezzo al tuo schermo per esempio '|cFF33FF99vault slot unlocked|r' verrano invece visualizzati come messaggi BigWigs. Questi banner possono essere abbastanza grandi, durare molto, e bloccare la possibilità di cliccarci attraverso."
```

Comment:
You missed the `vault slot unlocked` part, it should reflect the translation of the message you see in game.